src/main/java/Task.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * The Task class.
+ * Stores whatever tasks entered by the user and display them back to the user
+ * when requested.
+ * Acts as a superclass for all Task types.
+ */
+public class Task {
```

Comment:
Should this be abstract since there is never an instance of a task command?

Reply:
agreed